chain/cardano-sl-chain.cabal

```diff
@@ -275,7 +275,8 @@ test-suite chain-test
                        Test.Pos.Chain.Update.Gen
                        Test.Pos.Chain.Update.Json
 
-  build-depends:       base
+  build-depends:       aeson
```

Comment:
I like to keep `base` first because we'll almost certainly never move it, then we never get more than one line diff for an added package :)